mundane-io/src/main/scala/com/ambiata/mundane/io/LocalDirectory.scala

```diff
@@ -156,33 +159,27 @@ class LocalDirectory private (val path: Path) extends AnyVal {
 }
 
 object LocalDirectory {
-  def Root: Path =
-    com.ambiata.mundane.path.Root
-
-  def Relative: Path =
-    com.ambiata.mundane.path.Relative
-
-  def fromFile(f: File): LocalDirectory =
-    unsafe(f.getPath)
-
-  def fromString(s: String): Option[LocalDirectory] =
-    s.split("/").toList match {
-      case Nil =>
-        new LocalDirectory(Root).some
-      case "" :: Nil =>
-        None
-      case "" :: parts =>
-        parts.traverse(Component.create).map(fromList(Root, _))
-      case parts =>
-        parts.traverse(Component.create).map(fromList(Relative, _))
-    }
+  def fromFile(f: File): RIO[LocalDirectory] =
+    LocalPath.fromFile(f).flatMap(_.determineDirectory)
```

Comment:
Probably. It was copied from [LocalFile](https://github.com/ambiata/mundane/blob/master/mundane-io/src/main/scala/com/ambiata/mundane/io/LocalFile.scala#L230-L231) so should probably be changed there also